src/sentry/notifications/helpers.py

```diff
@@ -38,23 +39,49 @@
     )
 
 
-# This mapping represents how to interpret the absence of a DB row for a given
+# These mappings represents how to interpret the absence of a DB row for a given
 # provider. For example, a user with no NotificationSettings should be opted
 # into receiving emails but no Slack messages.
+NOTIFICATION_SETTING_DEFAULTS_DEFAULT = {
```

Comment:
can you add a comment here specifying this is for the A/B test? "defaults default" is kind of confusing